src/components/DimensionsPanel/List/__tests__/__snapshots__/DimensionItem.spec.js.snap

```diff
@@ -17,6 +17,7 @@ exports[`DimensionItem matches the snapshot 1`] = `
   }
 >
   <DimensionLabel
+    dataTest="undefined-button-pe"
```

Comment:
Did you intend to use the word "undefined" in this file?

Reply:
No, this is not intentional, but I think it comes from the component being tested without a parent or because the parent doesn't provide the dataTest prop. Either way I think this is fine since jest and cypress work in isolation.